projects/microphysics/configs/default.yaml

```diff
@@ -237,13 +237,13 @@ namelist:
     do_sat_adj: false
     nudge: false
     nudge_qv: false
-    warm_start: false
-    external_ic: true
-    external_eta: false
-    make_nh: true
-    nggps_ic: true
-    mountain: false
-    na_init: 1
+    warm_start: true
```

Comment:
I reset these to the previous configuration values as well, although, I do not know exactly which of them matter.